composer.json

```diff
@@ -85,6 +85,7 @@
 		"test-integration-adminonly": "\"vendor/bin/phpunit\" --testsuite integration --colors=always --configuration tests/Integration/phpunit.xml.dist --group AdminOnly",
 		"test-integration-docloudflare": "\"vendor/bin/phpunit\" --testsuite integration --colors=always --configuration tests/Integration/phpunit.xml.dist --group DoCloudflare",
 		"test-integration-withwoo": "\"vendor/bin/phpunit\" --testsuite integration --colors=always --configuration tests/Integration/phpunit.xml.dist --group WithWoo",
+		"test-integration-cpcss": "\"vendor/bin/phpunit\" --testsuite integration --colors=always --configuration tests/Integration/phpunit.xml.dist --group CriticalCss",
```

Comment:
Are you wanting the integration tests to run separately in our CI pipeline? Or is this a convenience script for running a group locally?

If it's a convenience script, you don't need it here. Instead, do this:

```
composer test-integration -- --group CriticalCss
```